amplify/backend/api/AdminQueries/cli-inputs.json

```diff
@@ -2,7 +2,7 @@
   "version": 1,
   "paths": {
     "/{proxy+}": {
-      "lambdaFunction": "AdminQueriesfd66d56a",
+      "lambdaFunction": "AdminQueries5b1ab4e5",
```

Comment:
This is fine, but I'd like to understand how these are named and updated better.

Reply:
It is a randomly generated string appended to AdminQueries to identify AdminQueries